scripts/visual-baselines.js

```diff
@@ -0,0 +1,71 @@
+/* jshint node: true */
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const rimraf = require('rimraf');
+const logger = require('@blackbaud/skyux-logger');
+
+const {
+  exec,
+  dirHasChanges
+} = require('./utils');
+
+const baselineScreenshotsDir = 'screenshots-baseline';
+const tempDir = '.skypagesvisualbaselinetemp';
+
+function handleBaselineScreenshots() {
+  const buildId = new Date().getTime();
```

Comment:
Can this be a relevant environment variable?  https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
My thought was to keep this generic for VSTS as well. Thoughts on how to accomplish that automatically?

Reply:
VSTS has similar variables.  https://docs.microsoft.com/en-us/vsts/pipelines/build/variables?view=vsts

Perhaps just an if statement to determine the ID or can this be baked in to the platform flag?

```
let buildId;
if (process.env.TRAVIS_BUILD_NUMBER) {
  buildId = process.env. TRAVIS_BUILD_NUMBER;
} else if (process.env.BUILD_BUILDID) {
  buildId = process.env.BUILD_BUILDID;
} else {
  buildId = new Date().getTime();
}
```